test/integration/lib/json/bridge_status.json

```diff
@@ -0,0 +1,445 @@
+{
+  "lights": {
+    "1": {
+      "state": {
+        "on": true,
+        "bri": 254,
```

Comment:
Maybe we could set the HSB values to easy-to-remember ones? Like (255, 16384, 255) ?